app/main.cpp

```diff
@@ -214,8 +215,28 @@ int main(int argc, char *argv[])
                                    QLatin1String("max-frames"), QLatin1String("64"));
     parser.addOption(maxFrames);
 
+    QCommandLineOption verbose(QLatin1String("verbose"),
```

Comment:
this should be it's own patch - it has nothing to do with the rest that's following below

Reply:
i.e. one patch for the verbose option
then one patch for the relAddr in the symbol
then one patch for the branchStack changes - what are they even doing?!